Ukoly/DU_6/Piskvorky.py

```diff
@@ -0,0 +1,74 @@
+from random import randrange
+
+WINNER = None
+
+
+def evaluation(string):
+    for _ in string:
+        if 'xxx' in string and 'ooo' not in string:
+            WINNER = '"x"'
+        elif 'ooo' in string and 'xxx' not in string:
+            WINNER = '"o"'
+        elif '-' in string and ('xxx' not in string or 'ooo' not in string):
+            WINNER = None
+        elif '-' not in string and \
+            ('xxx' not in string or 'ooo' not in string):
+            WINNER = 'no one'
+    if WINNER is not None:
```

Comment:
Toto je ok, ale možno by to chcelo dať do tej funkcie vyššie (tam kde voláš túto funkciu)... Ale to je len detail.